packages/bloc/pubspec.yaml

```diff
@@ -10,10 +10,11 @@ environment:
 
 dependencies:
   meta: ^1.1.6
-  rxdart: ^0.23.0
 
 dev_dependencies:
   test: ">=1.3.0 <2.0.0"
   test_coverage: ^0.2.0
   mockito: ^4.0.0
   effective_dart: ^1.2.0
+  rxdart: ^0.23.0
```

Comment:
Is there a reason to keep this as a `dev_dependency`? I think it might be worth to totally remove `rxdart` if possible!

Reply:
There are some tests that use rx operators (debounceTime, etc...)

Reply:
gotcha!